bigquery_storage/to_dataframe/main_test.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+
+import pytest
+
+
+@pytest.fixture
+def clients():
+    # [START bigquerystorage_pandas_tutorial_create_client]
+    import google.auth
+    from google.cloud import bigquery
+    from google.cloud import bigquery_storage_v1beta1
+
+    # Explicitly create a credentials object. This allows you to use the same
+    # credentials for both the BigQuery and BigQuery Storage clients, avoiding
+    # unnecessary API calls to fetch duplicate authentication tokens.
+    credentials, project_id = google.auth.default(
+        scopes=["https://www.googleapis.com/auth/cloud-platform"]
+    )
+
+    # Make clients.
+    bqclient = bigquery.Client(credentials=credentials, project=project_id)
+    bqstorageclient = bigquery_storage_v1beta1.BigQueryStorageClient(
+        credentials=credentials
+    )
+    # [END bigquerystorage_pandas_tutorial_create_client]
+    return bqclient, bqstorageclient, project_id
+
+
+@pytest.fixture
+def temporary_dataset(clients):
+    bqclient, _, _ = clients
+    dataset_id = "bqstorage_to_dataset_{}".format(uuid.uuid4().hex)
+
+    # [START bigquerystorage_pandas_tutorial_create_dataset]
+    from google.cloud import bigquery
+
+    # TODO: Set the dataset_id to the dataset used to store temporary results.
+    # dataset_id = "query_results_dataset"
```

Comment:
Nit (feel free to ignore): I think it would be clearer if you removed the TODO did something like this:

```python
dataset_id = "query_results_dataset"
# [END bigquerystorage_pandas_tutorial_create_dataset]
# Create a unique dataset ID for testing
dataset_id = "bqstorage_to_dataset_{}".format(uuid.uuid4().hex)
# [START bigquerystorage_pandas_tutorial_create_dataset]
```
(This would apply to all variable replacement TODOs in the tutorial)

Reply:
Done.